packages/web5/lib/src/crypto/key_managers/ios_key_manager.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:keychain/keychain.dart';
+import 'package:web5/web5.dart';
+
+class IosKeyManager implements KeyManager, KeyImporter, KeyExporter {
+  @override
+  Future<String> generatePrivateKey(AlgorithmId algId) async {
+    final Jwk privateKeyJwk = await Crypto.generatePrivateKey(algId);
+    final String alias = privateKeyJwk.computeThumbprint();
+
+    await Keychain.set(alias, json.encode(privateKeyJwk.toJson()));
+    return alias;
+  }
```

Comment:
@mistermoe and @StonePack do we want `web5-dart` to be flutter aware? I feel like given that it's a "pure dart" package we probably should avoid dependencies that require Flutter.

Maybe an alternative here is to keep this `IosKeyManager` out of the dart package completely and when it's needed for a Flutter project we could add a `web5-flutter` package.

Maybe this repo can have the structure:
```
- packages
  - web5-dart
  - web5-flutter (this one would have the iOSKeyManager and Keychain stuff)
```

Reply:
I think that's a great call, this key manager could easily be added to a web5-flutter package which can be used in tandem with the web5-dart package. I'll pull it out for now and if tbd wants to host a web5-flutter package I can create a separate PR or I'll just stash it until that time comes.

Reply:
I've removed the iOS key manager code and created a new repo as reference for that work: https://github.com/stonepack/web5-flutter